masterlist.yaml

```
@@ -14987,6 +15001,30 @@ plugins:
         util: '[SSEEdit v4.0.3g](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
         itm: 3
 
+  - name: 'HandtoHand.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/59790/' ]
+    tag:
+      - Names
+      - Text
+  - name: 'HandtoHandArmor.esp'
+    tag:
+      - Names
+      - Text
```

This plugin doesn't appear to change names beyond forwarding changes from its masters:
```suggestion
    tag: [ Text ]
```